### DISCUSSION TITLE
Attributing Scheduled Events to Users

### DISCUSSION THREAD

**Discussion by asluis3** [apache/airflow#34226] [Q&A]
Hi everyone,

I am trying to use Audit Logs to find out who (Airflow user) owns/is responsible for a scheduled dag run (and task instance). From what I'm able to see Airflow attributes the 'airflow' user as the user that scheduled those DAGs/Tasks instead of the person who unpaused the DAG. Would there be any way to get the information I need instead of maintaining state on who unpaused a DAG and attributing all subsequent dag runs to that user?

For context I am using the listener plugin to log these events, so maintaining state might not be the best approach unless I keep track of the unpause user in a database somewhere. Thank you!

(Using Airflow v2.5.3)

**Comment by potiuk** [comment]
You can read it from the Audit Log via API currently.
